variables.md

```diff
@@ -174,6 +174,7 @@ SES5_CEPH_QA_HEALTH_OK | string | | URL for repo containing ceph-qa-health-ok pa
 SKIP_CERT_VALIDATION | boolean | false | Enables linuxrc parameter to skip certificate validation of the remote source, e.g. when using self-signed https url.
 SET_CUSTOM_PROMPT | boolean | false | Set a custom, shorter prompt in shells. Saves screen space but can take time to set repeatedly in all shell sessions.
 SLE_PRODUCT | string | | Defines SLE product. Possible values: `sles`, `sled`, `sles4sap`. Is mainly used for SLE 15 installation flow.
+SLURM_VERSION | string | | Takes `slurm` to install current version, or expected version to be tested, ex 23_02. Use https://confluence.suse.com/pages/viewpage.action?pageId=1263338001 to identify version per SLE servive pack.
```

Comment:
Remove confluence reference from doc, its not reachable outside SUSE.
Also I wouls just use SLURM_VERSION with version and default value, if not specified should be slurm. It wil make more simple.

Reply:
i removed second sentence. review and close if all good